burstphoto/align.metal

```diff
@@ -13,30 +19,24 @@ kernel void fill_with_zeros(texture2d<float, access::write> texture [[texture(0)
 kernel void texture_uint16_to_float(texture2d<uint, access::read> in_texture [[texture(0)]],
                                     texture2d<float, access::write> out_texture [[texture(1)]],
                                     uint2 gid [[thread_position_in_grid]]) {
+    
     uint4 val_unint = in_texture.read(gid);
-    float4 val_float = float4(val_unint) / UINT16_MAX_VAL;
-    out_texture.write(val_float, gid);
-}
-
-
-kernel void add_texture(texture2d<float, access::read> texture1 [[texture(0)]],
-                        texture2d<float, access::read_write> texture2 [[texture(1)]],
-                        uint2 gid [[thread_position_in_grid]]) {
-    float4 out_color = texture1.read(gid) + texture2.read(gid);
-    texture2.write(out_color, gid);
+    
+    // removed scaling to the range 0 to 1 to improve fidelity of float16 values
```

Comment:
This comment belongs in the commit message or the PR, not in the code. In the code it does not add anything to the code that remains.

Reply:
That is true. Can be removed in a future version of the code.

Reply:
Added some information to the PR and removed the line of code in the commit [Improved documentation](https://github.com/chris-rank/hdr-plus-swift/commit/243113d9f16c35e8e035feb8b1529e656f96b6bf).